lib/URLHelper.php

```diff
@@ -292,6 +292,42 @@ public static function remove_trailing_slash( $link ) {
 	}
 
 	/**
+	 * Pass links through user_trailingslashit handling query strings properly
```

Comment:
@xavivars — if we were to use WP's `user_trailingslashit` w/o this it wouldn't handle query strings properly, do I have that right?